modules/extractTestLog.sh

```diff
@@ -1,6 +1,9 @@
 #!/bin/bash
 # Author  : 'BlackDChase'
-# Version : '1.5.3'
+# Version : '1.4.9'
+
+testVariant=$1
```

Comment:
I think I removed sharing `testVariant` via `run`. This might be the reason why it's causing bugs at your end. I think a `testVariant` was not required. If there will be a file it will act accordingly. Rethink the situation and refer to the code for `postTesting` as well before making a decision.